feature/encore-webpack (#5461)
sonata-project/SonataAdminBundle: docs/getting_started/installation.rst

    bin/console cache:clear
    bin/console assets:install

Webpack Encore Integration
--------------------------

> When the assets are installed add following to  your configuration:

Reviewer: ```suggestion
When the assets are installed add following to your configuration:
```

.. code-block:: yaml

>     # config/sonata_admin.yaml

Reviewer: ```suggestion
    # config/webpack_encore.yaml
```
?

Author: Why not putting this in sonata_admin for the sonata specific assets? see https://github.com/symfony/recipes-contrib/pull/714

    bin/console cache:clear
    bin/console assets:install

Webpack Encore Integration
--------------------------

> When the assets are installed add following to ``config/sonata_admin.yaml``

Reviewer 2: ```suggestion
When the assets are installed add following to  your configuration:
```

.. code-block:: yaml

> 

Reviewer 2: ```suggestion

    # config/sonata_admin.yaml
```

    webpack_encore:
        builds:
            sonata_admin: '%kernel.project_dir%/public/bundles/sonataadmin/dist'
            # Handling multiple entry points. For example app entry

>             #app: '%kernel.project_dir%/public/build'

Reviewer 2: I am for removing the comments

Author: How do we explain to someone who already has webpack? than he needs to do that.

Reviewer 2: why? he already has an entry with `app: ...` in another file and the get merged, right?

Author: Probably not because he uses the `assets` config section in `sonata_admin.yaml`

> This will load the assets to the template.

Reviewer 2: This sentence sounds a bit weird. Shall we reword or completely remove it?

Author: We can remove it.

> This will load the assets to the template.

Author: ```suggestion
```

    bin/console cache:clear
    bin/console assets:install

Webpack Encore Integration
--------------------------

> When the assets are installed add following to ``config/webpack_encore.yaml``

Reviewer 3: Maybe we could suggest to configure this in `config/sonata_admin.yaml` instead.

Reviewer 2: Yes 👍🏻

.. code-block:: yaml

    webpack_encore:
        builds:
            sonata_admin: '%kernel.project_dir%/public/bundles/sonataadmin/dist'
            # Handling multiple entry points. For example app entry
            #app: '%kernel.project_dir%/public/build'

This will load the assets to the template.

> 

Reviewer 3: Please, remove surplus LF.
